File: relatorio/codigos/ruby/micro06.rb
# Escreve um numero lido por extenso

puts("Digite um numero 1 a 5:")
numero = gets.chomp.to_i

case numero
when 1
  puts("Um")
when 2
  puts("Dois")
when 3
  puts("Tres")
when 4
  puts("Quatro")
when 5
  puts("Cinco")
else
  puts("Numero invalido!!!")
end

File: relatorio/codigos/lua/micro06.lua
-- Escreve um numero lido por extenso

--[[ Funcao: Faça um algoritmo que leia um número de 1 a 5 e o escreva por extenso. Caso o usuario digite um numero que nao esteja nesse intervalo, exibir mensagem: numero invalido --]]

print("Digite um número de 1 a 5")
numero = io.read("*number")
if(numero == 1) then print("Um")
elseif (numero == 2) then print("Dois")
elseif (numero == 3) then print("Três")
elseif (numero == 4) then print("Quatro")
elseif (numero == 5) then print("Cinco")
else print("Número Invalido!!!")
end